internal/uservalidator/user_validator.go

```diff
@@ -208,8 +236,19 @@ func (i *ValidateUser) Validate(ctx context.Context) ([]reconcile.ValidationErro
 		return nil, err
 	}
 
-	allValidationErrors = reconcile.ConcatValidationErrors(allValidationErrors, i.validateUsersSinglePath(*users))
-	allValidationErrors = reconcile.ConcatValidationErrors(allValidationErrors, i.validateUsersGithub(ctx, *users))
+	compareUsers, err := Users(context.WithValue(ctx, gql.UseCompareClientKey, true))
+	if err != nil {
+		return nil, err
+	}
+
+	if users == nil || compareUsers == nil {
+		return nil, fmt.Errorf("No users found")
+	}
```

Comment:
Do we always run it with `compareUsers`? How to get the full list?

Reply:
Yes, it's supposed to always run with compare users during MR check.
I wanted to get it running, afterwards I'll remove some of the concurrency checking and will add the pgp key checks again